testnets/berberistestnet/assetlist.json

```diff
@@ -0,0 +1,23 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "berberistestnet",
+  "assets": [
+    {
+      "description": "The native staking token of the Hedge blockchain.",
+      "denom_units": [
+        {
+          "denom": "hedge",
+          "exponent": 0
+        },
+        {
+          "denom": "uhedge",
```

Comment:
the denoms are backwards here. 'uhedge' should be the first denom with exponent 0, and the 'hedge' after with exponent 6

Reply:
Given that my base denom is defined as `hedge`, I believe it is okay to assign an exponent of 0 to `hedge` and an exponent of 6 to `uhedge`.

Reply:
I understood where I might have gone wrong in my denom_metadata. I will get on to that and update in my genesis. If I do that the following changes makes no sense and I need to roll back to how it was earlier except the denom_metadata. Will update in our chain as well.

Reply:
This should resolve our confusions. https://github.com/cosmos/chain-registry/pull/3806/commits/2d1ac1c73951bb89aa033fb4a57e5cf4dbb4300a